### PR TITLE
docs(cli): Add v0.2.6 release notes

### PR DESCRIPTION
**Pull request by conneryn**
## Description

Update release notes for v0.2.6 of the CLI.

### CODE REVIEW

File: content/changelogs/cli.md [hunk 1]
```diff
@@ -3,7 +3,31 @@
...
 > [!WIP]
-> The CLI is currently in an alpha pre-release state, and may include breaking changes between releases.
+> The CLI is currently in an alpha state, and may include breaking changes between releases.
+
+## Release Notes for v0.2.6
+
+### Bug Fixes
+
+* Fix error messaging to users of unsupported Node versions (<= 12.x)
+* Fixed several `czctl deployment` arguments not being properly recognized [#1834, #1837]
```
Comment (nsainaney) [inline content/changelogs/cli.md:13]: Some not several

File: content/changelogs/cli.md [hunk 2]
```diff
@@ -3,7 +3,31 @@
...
+
+* Teleport provides feedback to the user once all connections are fully setup [#1766, #1827]
+* Teleport auto-detects when new intercepts are added/removed, so restart is no longer required [#1808]
+* Extended teleport to work with other workloads (pods, cronjobs, jobs, statefulsets) [#1778]
+
+### Outstanding Issues
+
+* NodesJS <= 12.x is not supported.
+* Cannot intercept more than 4 services simultaneously from the same machine.
+* Cannot teleport to an external service (workload must have an in-cluster pod)
```
Comment (nsainaney) [inline content/changelogs/cli.md:25]: I think you mean headless service intercepts

**Review by gburnard — APPROVED** [review]
Looks fine from my persective.

**Review by robblovell — APPROVED** [review]

File: content/changelogs/cli.md (hunk 2 above)
Comment (conneryn) [reply content/changelogs/cli.md:25]: My understanding was this is just related to teleport (not intercept...)?   

But I agree I should use the terminology charlieAI is already using/expecting, so I'd propose:
```suggestion
* Cannot teleport to headless services yet
```

File: content/changelogs/cli.md (hunk 1 above)
Comment (conneryn) [reply content/changelogs/cli.md:13]:
```suggestion
* Fixed some `czctl deployment` arguments not being properly recognized [#1834, #1837]
```

**Pull request merged**